app/templates/application.hbs

```diff
@@ -2,7 +2,7 @@
 {{google-jsapi}}
 
 <a href='https://github.com/rust-lang/crates.io' class='fork-me'>
-  <img src='/assets/forkme.png'/>
+  <img src='/assets/forkme.png'/ alt="Fork me on GitHub">
```

Comment:
I think the slash after the `src` attribute is to close the tag, so should go after the `alt` attribute.